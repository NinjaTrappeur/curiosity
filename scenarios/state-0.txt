reset
run init-users.txt
run init-entities.txt
run init-units.txt
