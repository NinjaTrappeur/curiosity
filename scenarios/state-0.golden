1: reset
Resetting to the empty state.
2: run init-users.txt
> 2: user create alice a alice@example.com --accept-tos
> User created: USER-1
> 3: user update USER-1 Alice Hi ! I'm the first advisor registered on this prototype. I know and have access to everything... Let me know if you need anything !
> User updated: USER-1
> 5: user create bob b bob@example.com --accept-tos
> User created: USER-2
> 6: user create charlie c charlie@example.com --accept-tos
> User created: USER-3
> 8: user create mila m mila@example.com --accept-tos
> User created: USER-4
> 9: user update USER-4 Mila I'm Mila, the first member to join this prototype. My advisor is @alice and I've created @alpha.
> User updated: USER-4
3: run init-entities.txt
> 1: entity create one One S.A. 100200300 BE0100200300
> Legal entity created: LENT-1
> 2: entity update one One S.A. is the main legal entity of the prototype. Unless required for fiscal or legal reasons, new contracts are hosted in One.
> Legal entity updated: one
> 3: entity link-user one USER-1 --validator
> Legal entity updated: one
> 4: entity set-supervised one
> Legal entity updated: one
> 5: entity set-host one
> Legal entity updated: one
> 7: entity create two Two S.A. 100200400 BE0100200400
> Legal entity created: LENT-2
> 8: entity update two Two S.A. is the second legal entity of the prototype.
> Legal entity updated: two
> 9: entity link-user two USER-1 --validator
> Legal entity updated: two
> 10: entity set-supervised two
> Legal entity updated: two
> 11: entity set-host two
> Legal entity updated: two
4: run init-units.txt
> 1: unit create alpha Alpha
> Business entity created: BENT-1
> 2: unit update alpha The first business unit of the prototype, Alpha is run by @mila.
> Business unit updated: alpha
> 3: unit link-user alpha USER-4 --holder
> Business unit updated: alpha
5: run init-form-simple-contracts.txt
> 1: as mila
> Modifying default user.
> 2: forms simple-contract new --amount -100
> Simple contract form created: TBPJLIUG
> 3: forms simple-contract new
> Simple contract form created: HNONWPTG
> 4: forms simple-contract new --amount 100
> Simple contract form created: RZEMQMNF
